binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -97,6 +95,12 @@ public void initialize() {
             // note: this call implicitly registers our handler as a listener on the bridge
             if (getHueBridgeHandler() != null) {
                 getThing().setStatus(getBridge().getStatus());
+                FullLight fullLight = getLight();
+                if (fullLight != null) {
+                    //TODO persist changes if ticket# is available
```

Comment:
Could you please add the concrete ticket with the link number here:  https://bugs.eclipse.org/bugs/show_bug.cgi?id=461874